subsys/net/ip/net_context.c

```diff
@@ -886,6 +886,64 @@ static int create_udp_packet(struct net_context *context,
 }
```

Comment:
In patch "net: context: Split packet sending into two functions", typo in commit message s/definitio/definition/

Reply:
Fixed.